enterprise/backend/test/metabase_enterprise/serialization/v2/models_test.clj

```diff
@@ -48,34 +49,38 @@
 (deftest serialization-complete-spec-test
   (mt/with-empty-h2-app-db
     ;; When serialization spec is defined, it describes every column
-    (doseq [m     serdes.models/exported-models
-            :let  [spec (serdes/make-spec m)]
+    (doseq [m     (-> (methods serdes/make-spec)
+                      (dissoc :default)
+                      keys) #_ serdes.models/exported-models
```

Comment:
There should still be a test for models in `serdes.models/exported-models` that do not have implementations for `serdes/make-spec`